lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -291,7 +303,8 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
-    private Optional<InheritedIdentityJwtClaim> getInheritedIdentityClaim(JWTClaimsSet claimsSet)
+    private Optional<ListOfStringValues> getStringListClaim(
```

Comment:
I don't really like this `isForInheritedIdentityClaim` argument - how would this work if we added another claim?

The interesting bit of this method is the parsing of the claims, so perhaps we should reduce this method down to `getJarClaims`, and then access the other bits where they're needed?

e.g. `getJarClaims(claimsSet).userInfo().inheritedIdentityClaim()` (being careful to avoid nulls where they may occur)